### PR TITLE
improvements: Added conditions in dynamic CSS to load when component is enabled

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Added conditions for sidebar dynamic CSS to load when it is enabled.
- Also separate out static sidebar CSS to PHP file and load conditionally,
- Added RTL support conditions for respect static CSS.

### Types of changes
- improvements

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards

### CODE REVIEW

**Review by shubhw12 — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

File: inc/class-astra-dynamic-css.php [hunk 1]
```diff
@@ -1006,48 +1047,44 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
...
+						),
+						'.ast-page-builder-template.ast-right-sidebar #secondary' => array(
+							'padding-left' => '20px',
+						),
+						'.ast-right-sidebar #secondary' => array(
+							'padding-left' => 0,
+						),
+						'.ast-left-sidebar #primary'    => array(
+							'padding-left' => 0,
+						),
```
Comment (imnavanath) [inline inc/class-astra-dynamic-css.php:1073]: Let's merge the above selector here - '.ast-right-sidebar #secondary'  as this has same CSS padding-left: 0

File: inc/class-astra-dynamic-css.php [hunk 2]
```diff
@@ -2589,6 +2616,69 @@ public static function gutenberg_media_text_block_css_compat() {
...
 
+		/**
+		 * Load sidebar static CSS when it is enabled.
+		 *
+		 * @since 3.0.0
+		 */
+		public static function load_sidebar_static_css() {
+
+			return '
+			#secondary {
```
Comment (imnavanath) [inline inc/class-astra-dynamic-css.php:2627]: How we are managing the rtl() case here ?? Need to check is_rtl() case here. Let's have call if needed,

File: inc/addons/breadcrumbs/dynamic-css/dynamic.css.php [hunk 3]
```diff
@@ -27,6 +27,10 @@ function astra_breadcrumb_section_dynamic_css( $dynamic_css, $dynamic_css_filter
 
 	$breadcrumb_position = astra_get_option( 'breadcrumb-position', 'none' );
 	
+	if ( 'none' === $breadcrumb_position ) {
+		return $dynamic_css;
```
Comment (imnavanath) [inline inc/addons/breadcrumbs/dynamic-css/dynamic.css.php:31]: Breadcrumb shortcode will not work, as the following CSS is important for breadcrumb.
with short code - https://a.cl.ly/P8umXljA
in this branch - https://a.cl.ly/RBu9J0w1

Comment (sagarb90) [reply inc/addons/breadcrumbs/dynamic-css/dynamic.css.php]: @imnavanath As per discussion revert this condition.

**Review by imnavanath — APPROVED** [review]

**Pull request merged**